lib/chrome.js

```diff
@@ -211,32 +201,59 @@ class Chrome extends EventEmitter {
         }
     }
 
-    // establish the WebSocket connection and start processing user commands
-    _connectToWebSocket() {
+    _createStdioWrapper() {
+        const stdio = new StdioWrapper(this.process.stdio[3], this.process.stdio[4]);
```

Comment:
i also considered having the user pass the `writeStream` and `readStream` directly, but figured that there is not a use case for it